example/gluon/style_transfer/main.py

```diff
@@ -0,0 +1,214 @@
+import time
+import random
+import os
+import mxnet as mx
+import numpy as np
+np.set_printoptions(precision=2)
+from PIL import Image
+
+from mxnet import gluon
+from mxnet.gluon import nn, autograd, Block, HybridBlock, Parameter, ParameterDict
+import mxnet.ndarray as F
+
+import net
+import utils
+from option import Options
+
+def train(args):
+    # temporary solutution for dataloader
+    from torchvision import datasets
+    from torchvision import transforms
+
+    np.random.seed(args.seed)
+    if args.cuda:
+        ctx = mx.gpu(0)
+    else:
+        ctx = mx.cpu(0)
+    # dataloader
+    transform = transforms.Compose([transforms.Scale(args.image_size),
+                                    transforms.CenterCrop(args.image_size),
+                                    utils.ToTensor(ctx),
+                                    ])
+    train_dataset = datasets.ImageFolder(args.dataset, transform)
+    train_loader = gluon.data.DataLoader(train_dataset, batch_size=args.batch_size, last_batch='discard')
+    style_loader = utils.StyleLoader(args.style_folder, args.style_size, ctx=ctx)
+    print('len(style_loader):',style_loader.size())
+    # models
+    vgg = net.Vgg16()
```

Comment:
This has been addressed here:
https://github.com/apache/incubator-mxnet/pull/7579/commits/5ae3588d4fae64593f2c9483a02f864540ceddbd#diff-568c794d7c719a2408f5e7c8bbdc17e0